CHANGELOG.md

```diff
@@ -17,8 +17,3 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 [Unreleased]: <https://github.com/stac-extensions/template/compare/v1.0.0...HEAD>
-
-## [v1.0.0-beta.1] - 2020-05-29
```

Comment:
You either need to use v1.0.0 or change the URLs pointing to v1.0.0 in the README and potentially other places.